packages/console-sdk-microfrontend/package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "@mia-platform/microfronent-sdk",
+  "name": "@mia-platform/console-sdk-microfrontend",
   "license": "Apache-2.0",
   "version": "0.0.0",
   "private": true,
-  "main": "./build/src/index.js",
+  "main": "./src/index.ts",
```

Comment:
Why not the build? By default, we use the non compiled lib?

Reply:
Local work is very struggle because every time we change "microfrontend-sdk" we need to rebuild packages. I the same time the import from other packages is broke up with errors in the missing repository. 

To fix them we need to restart the typescript server with cmd+shift+p.

Reply:
Is correct to use "build/src/index.js", but the package.json module seems not to work as we think

Reply:
I try to fix the annoying problem of the server following this guide.
https://pnpm.io/package_json#publishconfig

So I use internal index.ts and override this information on the build. 
I try to generate with pnpm pack result but I see the same package.json ....

Package.json of each workspace repo
```
"main": "./src/index.ts",
  "publishConfig": {
    "main": "./build/src/index.js",
    "typings": "./build/src/index.d.ts"
  },
```